audio-buffer-source-node/loop/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+
+    <title>Web Audio API examples: looping a track</title>
+
+    <script src="script.js" defer></script>
+  </head>
+
+  <body>
+    <h1>Web Audio API examples: looping a track</h1>
+
+    <button id="play" disabled>Play</button>
+    <button id="stop">Stop</button>
```

Comment:
Small change related to the change in where AudioContext is created in the script.js.
```suggestion
    <button id="play>Play</button>
    <button id="stop" disabled>Stop</button>
```